BookingsAPI/Bookings.IntegrationTests/Steps/ParticipantsSteps.cs

```diff
@@ -92,7 +92,7 @@ public async Task WhenISendTheSameRequestButWithANewHearingId()
             Context.TestData.OldHearingId = Context.TestData.NewHearingId;
             Context.TestData.NewHearingId = seededHearing.Id;
             NUnit.Framework.TestContext.WriteLine($"New seeded video hearing id: {seededHearing.Id}");
-            seededHearing.GetParticipants().Count.Should().Be(4);
+            seededHearing.GetParticipants().Count.Should().Be(5);
```

Comment:
Rather than using a hardcoded value, it would be safer to track the current or previous count and ensure it's higher than that.

Reply:
The amount of participants shouldn't really change, perhaps this should be in a variable instead. It is increased to 5 as part of this PR because we are adding another individual participant to the hearing to act as an interpreter

Reply:
Changed to a variable